packages/theme-chalk/src/button.scss

```diff
@@ -40,7 +40,9 @@ $button-icon-span-gap: map.merge(
   vertical-align: middle;
   -webkit-appearance: none;
 
-  &:not(.is-text):not(.is-link):not(.#{$namespace}-button--text) {
+  &:not(.is-text),
```

Comment:
this has the same effect with remove `:not`, maybe we can consider remove `:not()`.

Reply:
If  remove `:not()`, the basic usage seems to lose the style.

Reply:
```diff
- &:not(.is-text):not(.is-link):not(.#{$namespace}-button--text) { // line 43

- } // line 68
```

I tried it and didn't find the error you said

Reply:
I seem to have misunderstood what you meant. Did you mean to delete lines 43 to 68?

Reply:
just 43 and 68

Reply:
I tried it and it looks very good 👍